hack/tools/go.mod

```diff
@@ -3,29 +3,14 @@ module sigs.k8s.io/cluster-api-provider-aws/hack/tools
 go 1.13
 
 require (
-	cloud.google.com/go v0.75.0 // indirect
```

Comment:
I'm curious. How does this fix the issue? The commit message nor PR description does not say anything.